source/dapp.rst

```diff
@@ -3,10 +3,26 @@
 Credits Dapp
 ============
 
-Credits Dapp is a packaged Python module code that provides a
+Credits Dapp is a distributed blockchain-based application developed using
+the Credits Core framework. Dapp is a unit of business logic, and it's
+supposed to be designed with specific real life use case in mind.
+
+Credits Dapp is a software module designed to run as part of the Credits
+Core node, it's not a standalone application in itself. However it's not
+meant to be uploaded into the blockchain as a smart contract code. Instead it
+is provided as a part of network setup and configuration before Core node is
+started. Once the node is started the Dapp runs as integral part of the
+Core, inside Core's runtime.
+
+Credits Dapp is a Python module, and has to be discoverable to the Core
```

Comment:
In this context "dapp" is a subset of "module". It is everything module is, but gives more specifics. I'd prefer to leave it a `dapp create`.